extensions/machine-learning-services/package.json

```diff
@@ -110,7 +114,7 @@
                   "mls.command.managePackages",
                   "mls.command.manageLanguages",
                   "mls.command.manageModels",
-                  "mls.command.registerModel"
+                  "mls.command.predictModel"
```

Comment:
not in scope for this PR but just to note, we should strart trying to get icons for task widget items on dashboard as they will move to toolbar and wont look good without icons.

Reply:
requested for icons already

---
In reply to: [389913329](https://github.com/microsoft/azuredatastudio/pull/9450#discussion_r389913329) [](ancestors = 389913329)